ignite/metrics/classification_report.py

```diff
@@ -0,0 +1,107 @@
+import json
+from typing import Callable, Collection, Dict, List, Optional, Union
+
+import torch
+
+from ignite.metrics.fbeta import Fbeta
+from ignite.metrics.metric import Metric
+from ignite.metrics.metrics_lambda import MetricsLambda
+from ignite.metrics.precision import Precision
+from ignite.metrics.recall import Recall
+
+__all__ = ["ClassificationReport"]
+
+
+def ClassificationReport(
+    beta: int = 1,
+    output_dict: bool = False,
+    output_transform: Optional[Callable] = None,
```

Comment:
Let's use `output_transform` same as default one for `Metric` -> we can simply pass it below without
```python
output_transform=(lambda x: x) if output_transform is None else output_transform
```